src/redux/reducers/reportReducer.js

```diff
@@ -0,0 +1,36 @@
+import {
+  ARTICLE_REPORTED,
+  REPORT_ERROR,
+  REPORTING_ARTICLE
+} from "../actionTypes";
+
+const initialState = {
+  response: "",
+  reporting: false
+};
+export const reportArticle = (state = initialState, action) => {
+  switch (action.type) {
+    case ARTICLE_REPORTED:
+      return {
+        ...state,
+        response: action.payload,
+        reporting: false
+      };
+    case REPORT_ERROR:
+      return {
+        ...state,
+        response: action.payload,
+        reporting: false
+      };
+    case REPORTING_ARTICLE:
+      return {
+        ...state,
+        reporting: true
+      };
+    default:
+      return {
+        ...state,
```

Comment:
Can I know why you are returning both state and initial state here?
It looks like there is a duplication...